x/stake/keeper/validator.go

```diff
@@ -191,9 +191,9 @@ func (k Keeper) ClearTendermintUpdates(ctx sdk.Context) {
 
 //___________________________________________________________________________
 
-// Perfom all the nessisary steps for when a validator changes its power.  This
+// Perfom all the necessary steps for when a validator changes its power. This
 // function updates all validator stores as well as tendermint update store.
-// It may kick out validators if new validator is entering the bonded validator
+// It may kick out validators if a new validator is entering the bonded validator
```

Comment:
I didn't think it was worth the overhead of making a separate PR for these typo fixes, so I just included it here.